src/pods/test-verb-forms/components/show-results.component.tsx

```diff
@@ -9,17 +10,65 @@ interface Props {
 }
 
 export const ShowResults: React.FC<Props> = props => {
```

Comment:
Let's componentize this, ShowResults should look like:

Pseudocode not passing all the props down to the components (please include props :))
```tsx
export const ShowResults: React.FC<Props> = props => {
return verbCorrect.all ? (
  <ShowResultCorrect/>
) 
: (
 <ShowReusltFailed/>
)
}
```